### PR TITLE
catalog,po: update Polish translation 230528

### PR DESCRIPTION
**Pull request by piotrdrag**

### CODE REVIEW

File: catalog/systemd.pl.catalog.in [hunk 1]
```diff
@@ -542,3 +542,39 @@ Support: %SUPPORT_URL%
 
 Po raz pierwszy podczas obecnego uruchomienia uzyskano synchronizację NTP
 i zainicjowano regulację lokalnego zegara systemowego.
+
+-- 7db73c8af0d94eeb822ae04323fe6ab6
+Subject: Początkowe zwiększenie zegara
+Defined-By: systemd
+Support: %SUPPORT_URL%
+
+Zwiększono zegar systemowy na podstawie pliku ze znacznikiem czasu na dysku
```
Comment (keszybz) [inline catalog/systemd.pl.catalog.in]: Zwiększono → Przestawiono?

File: catalog/systemd.pl.catalog.in [hunk 2]
```diff
@@ -542,3 +542,39 @@ Support: %SUPPORT_URL%
...
+Rejestr konfiguracji platformy (PCR) @PCR@ układu TPM, w rzędach @BANKS@,
+został rozszerzony ciągiem „@MEASURING@”.
+
+Kiedy system przechodzi do nowego etapu uruchamiania, podany PCR jest
+rozszerzany innym ciągiem, aby zapewnić, że zasady zabezpieczeń dla haseł
+związanych z TPM i innych zasobów są ograniczone do konkretnych etapów
+uruchamiania.
+
+-- f9b0be465ad540d0850ad32172d57c21
+Subject: Obniżono pamięć
```
Comment (keszybz) [inline catalog/systemd.pl.catalog.in]: "Obniżono pamięć" → "Obniżono zużycie pamięci" ?

File: catalog/systemd.pl.catalog.in (hunk 1 above)
Comment (piotrdrag) [reply catalog/systemd.pl.catalog.in]: “Bump” suggests to me an increase of some sort, but if that’s not important to preserve, we can go with your suggestion.

File: catalog/systemd.pl.catalog.in (hunk 2 above)
Comment (piotrdrag) [reply catalog/systemd.pl.catalog.in]: Sure, that works.

File: catalog/systemd.pl.catalog.in (hunk 1 above)
Comment (keszybz) [reply catalog/systemd.pl.catalog.in]: "Bump" is a bit sloppy, but OK. But in Polish, you would never say "zwiększono zegar", I think.

**Comment by piotrdrag** [comment]
@keszybz In case you missed it, I applied your suggestions.

**Pull request merged**